config/settings.yml

```diff
@@ -97,6 +97,7 @@
     :purge_window_size: 10000
 :ems:
   :ems_redhat:
+    :resolve_ip_addresses: true
```

Comment:
@jhernand @adahms thanks for working this out and making sure it gets proper attention from docs.
I think its good to go now 👍